dashboard/lib/services/i18n/curriculum_sync_utils/sync_out.rb

```diff
@@ -170,13 +170,15 @@ def self.postprocess(result)
         def self.flatten(object_hash, serializer, name)
           results = {}
           object_hash.each do |key, object|
-            object = object.symbolize_keys
+            if object.is_a?(Hash)
```

Comment:
Does this mean the content is in the wrong format?
Should we log a warning?
Should we instead handle the string case? `results[name][key] = object if object.is_a?(String)`

Reply:
This is a good question. When I was investigating the issue, the String case I saw was a subset of a different object hash (e.g. one object was configured as `{'key': 'key', 'object': {'foo': 'bar'}}` before symbolizing keys, and the next was configured as `{'foo': 'bar'}`). Based on that example, I made the (maybe premature) assumption that the String case is redundant. I am on board with adding a warning though!